etcdserver/etcdserverpb/rpc.proto

```diff
@@ -438,6 +440,33 @@ message DefragmentResponse {
   ResponseHeader header = 1;
 }
 
+enum AlarmType {
+	NONE = 0; // default, used to query if any alarm is active
+	NOSPACE = 1;
+}
+
+message AlarmRequest {
+  // MemberID is the member making the alarm request
+  int64 memberID = 1;
```

Comment:
doc that only activate and deactivate needs memberID? probably move memberID to number 2?